tests/test_dbapi.py

```diff
@@ -112,20 +112,23 @@ def test_kdf_iter_is_sent(self, cursor, query_string, festung):
 
 class TestMultipleResponses(object):
     def test_executemany(self, festung, cursor):
-        for _ in range(3):
-            festung.add_json_response(
-                {'data': [], 'headers': [], 'last_row_id': 0, 'rows_changed': 3})
-
         query = 'UPDATE TABLE foo SET foo=?, bar=?, baz=?'
         params_list = [
             ['abc', 1, None],
             ['def', 2, 1.2],
             ['ghi', 3, None],
         ]
+        n_rows_changed = 3
```

Comment:
Maybe that's a stretch, but the test still pass if I do `self._rowcount = len(parameter_sequence) * rowcount`.

One way to make sure this test doesn't pass is to do this:

```
row_changed = [7, 8, 9]
for n_row_changed in row_changed:
    festung.add_...

assert cursor.rowcount == sum(row_changed)
```